pmlc/.clang-format

```diff
@@ -1,2 +1,2 @@
 BasedOnStyle: LLVM
-AlwaysBreakTemplateDeclarations: Yes
+AlwaysBreakTemplateDeclarations: true
```

Comment:
This file came directly from upstream: https://github.com/plaidml/llvm-project/blob/plaidml/plaidml-v1/mlir/.clang-format

I wonder if you're using an older version of clang-format? We use version 9.

Reply:
I see. I upgraded clang-format and this issue went away.